src/storage.test.ts

```diff
@@ -0,0 +1,156 @@
+import { storage } from './storage';
+
+const testStorage = (
+	storageName: 'localStorage' | 'sessionStorage',
+	// eslint-disable-next-line @typescript-eslint/no-explicit-any
+	fn: any,
```

Comment:
Can this be `fn: () => unknown`?

Reply:
I tried unknown and a few more besides but I ended up starting to build a type for the entire storage object, which felt excessive for a test, so I stopped, and did the dirty